pkg/kubelet/cm/helpers_linux_test.go

```diff
@@ -872,3 +872,58 @@ func TestResourceConfigForPodWithEnforceMemoryQoS(t *testing.T) {
 		}
 	}
 }
+
+func TestIsRewrite(t *testing.T) {
+	tests := []struct {
+		name     string
+		previous string
+		current  string
+		expected bool
+	}{
+		{
+			name:     "Empty previous",
+			previous: "",
+			current:  "/sys/fs/cgroup/memory",
+			expected: true,
+		},
+		{
+			name:     "previous same with current",
+			previous: "/sys/fs/cgroup/memory",
+			current:  "/sys/fs/cgroup/memory",
+			expected: false,
+		},
```

Comment:
Please, add one more test case with equal previous and current that both don't start with the "/sys/fs/cgroup" prefix.

Reply:
I don't think we need this test since the first IF condition  `if previous == current {` does not care the prefix and now it's covered